apex_rostest/apex_launchtest_ros/apex_launchtest_ros/data_republisher.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class DataRepublisher:
+    """Republish mesasges with a transform function applied."""
+
+    def __init__(self, node, listen_topic, publish_topic, msg_type, transform_fn):
+        """
+        Create a DataRepublisher.
+
+        :param node: A rclpy node that will run the publisher and subscriber
```

Comment:
nitpick: Nodes don't run anything...

Reply:
Yeah, this could be better - what's the approved verb here?  "Host" "Manage" something else?

Reply:
I'll fix it up on the next PR. I wanted to pull all commits as-they-are upstream.

Reply:
"create", I'd say.

It's fine to fix them up later (that's why I approved as well).